src/Score.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
+
+function Score() {
+ this.runningScore = 0;
+ this.strike = false;
+ this.spare = false;
+}
+
+Score.prototype.calculateScore = function (frameScore) {
+  if(this.strike === true) {
+    this.runningScore += ((2 * frameScore[0]) + (2 * frameScore[1]));
+  } else if (this.spare === true) {
+    this.runningScore += ((2 * frameScore[0]) + frameScore[1]);
+  } else {
+    this._checkForStrike(frameScore);
+    this._checkForSpare(frameScore);
+    this.runningScore += (frameScore[0] + frameScore[1]);
+    return frameScore[0] + frameScore[1];
+  }
+};
+
+Score.prototype._checkForStrike = function (frameScore) {
```

Comment:
Could the `Score` object provide a method that returns whether it represents a strike, rather than calculating that in advance and then exposing the state?